Hymns/Common/CustomTitleView.swift

```diff
@@ -0,0 +1,22 @@
+import SwiftUI
+
+struct CustomTitle: View {
+    var title: String
+
+    init(title: String) {
+        self.title = title
+    }
+
+    var body: some View {
+        HStack {
+            Text(self.title).customTitleLayout()
+            Spacer()
+        }
+    }
+}
+
+struct CustomTitle_Previews: PreviewProvider {
+    static var previews: some View {
+        CustomTitle(title: "Fav")
```

Comment:
You don't have to, but if you did like a 'CustomTitle(title: "Favorites").previewLayout(.fixed(width: 200, height: 50))`, it's a little bit nicer on preview